src/main/resources/com/microsoft/azure/vmagent/AzureVMComputer/configure.jelly

```diff
@@ -3,22 +3,23 @@
   <l:layout permission="${it.EXTENDED_READ}" title="${it.name} Configuration">
     <st:include page="sidepanel.jelly"/>
     <l:main-panel>
+      <j:set var="instance" value="${it.node}" />
+      <j:set var="descriptor" value="${instance.descriptor}" />
+      <j:set var="readOnlyMode" value="true" />
```

Comment:
cleans up the UI by using read only controls for everything on the page.

In the earlier version the node properties appeared to be modifiable but we don't want that